site/zenodo_rdm/migrator/extract.py

```diff
@@ -0,0 +1,19 @@
+import json
+
+from invenio_rdm_migrator.extract import Extract
+
+
+class JSONLExtract(Extract):
```

Comment:
minor: could this be part of `invenio_rdm_migrator`?

Reply:
I decided not to since it is slightly custom due to our dump. We are not using jsonlines but doing the custom replace and then loading the JSON. Therefore, it would only work with a specific type of `""` input.